sections/semantics-document-metadata.include

```diff
@@ -839,11 +837,9 @@
   </pre>
   </div>
   
-  <p>The use of these values could be appropriate in specific use cases, for example in a map application where 
-  zooming is handled via scripting or for a game application where a fixed viewport is required. 
-  In general however, their use will not be appropriate and HTML conformance checking tools should 
-  display a warning if developers use these values.</p>
-  
+  <p>There may be specific use cases where preventing users from zooming may be appropriate, such as map applications (where custom zoom functionality is handled via scripting) or games (designed specifically for a fixed viewport). However, in general this practice should be avoided, and HTML conformance checking tools should display a warning if they encounter these values.</p>
+
```

Comment:
What I said before. Design for a fixed viewport is too broad a statement IMHO.

Where there *is* a zoom mechanism, handled by the application - as in the map example - there is a clear use case. Can you explain the game case more clearly?